docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,10 @@ develop
     *    - Type
          - Change
 
+    *    - |fixed|
+         - Targeted sweep no longer chokes if the table for which a write it is trying to sweep no longer exists.
```

Comment:
-> "Targeted sweep no longer chokes if a table in the queue no longer exists" is clearer I think